pkg/services/sqlstore/sqlstore.go

```diff
@@ -99,6 +99,8 @@ func (ss *SqlStore) Init() error {
 	// Register handlers
 	ss.addUserQueryAndCommandHandlers()
 
+	ss.skipEnsureAdmin = setting.DisableAdminUser
```

Comment:
You shouldn't use the global setting here. Use `ss.skipEnsureAdmin = ss.Cfg.DisableAdminUser`.

Reply:
You probably need to change code in setting package and move `DisableAdminUser` to "cfg" struct.

Reply:
Think you can remove `ss.skipEnsureAdmin` alltogether and check only the `DisableAdminUser` setting.